index.html

```diff
@@ -1,225 +1,320 @@
+            <img
+              class="down_arrow_icon"
+              src="./images/Icon - Chevron - Down - Larger.png"
+              alt="down"
+            />
+          </h3>
+          <div class="frame-work_items">
+          <li class="ul_item">
+            <img src="./images/Ellipse 1.png" alt="Javascript" /><span
+              >Javascript</span
             >
           </li>
           <li class="ul_item">
             <img src="./images/Ellipse 1 (1).png" alt="HTML" /><span>HTML</span>
-                </li>
-                <li class="ul_item">
-                    <img src="./images/Ellipse 1 (2).png" alt="CSS" /><span>CSS</span>
-                </li>
-            </ul>
-            <ul class="about_ul">
-                <h3 class="sub_title">
-                    Frameworks
-                    <img class="arrow_icon" src="./images/Icon - Chevron - Right - Larger.png" alt="down" />
-                </h3>
-            </ul>
-            <ul class="about_ul">
-                <h3 class="sub_title">
-                    Skills
-                    <img class="arrow_icon" src="./images/Icon - Chevron - Right - Larger.png" alt="down" />
-                </h3>
-            </ul>
-        </section>
-        <section class="contact_form">
-            <h2 class="Title">Contact me</h2>
-            <p class="CONTACT_form">
-                If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it
-            </p>
-            <form class="form_container" action="https://formspree.io/f/mnqroaaw" method="POST">
-                <fieldset class="form_inputs">
-                    <input class="form_input" name="name" type="text" placeholder="Yeremias" maxlength="30" required />
-                    <input class="form_input" name="email" type="email" placeholder="natajayanj14@gmail.com" required />
-                    <textarea class="form_text_area" name="message" placeholder="Write your message here" maxlength="500" required></textarea>
-                    <button type="submit" class="btn_submit">Get It Touch</button>
-                </fieldset>
-            </form>
-        </section>
+          </li>
+          <li class="ul_item">
+            <img src="./images/Ellipse 1 (2).png" alt="CSS" /><span>CSS</span>
+          </li>
+          </div>
+        </ul>
+        <ul class="about_ul">
+          <h3 class="sub_title">
+            Frameworks
+            <img
+              class="arrow_icon"
+              src="./images/Icon - Chevron - Right - Larger.png"
+              alt="down"
+            />
+          </h3>
+        </ul>
+        <ul class="about_ul">
+          <h3 class="sub_title">
+            Skills
+            <img
+              class="arrow_icon"
+              src="./images/Icon - Chevron - Right - Larger.png"
+              alt="down"
+            />
+          </h3>
+        </ul>
+        </div>
+      </section>
```

Comment:
- [ ] You also need to work on this section of your project in order to produce the exact design of what we have on the Figma template. Consider giving your about section margin or padding to the both sides and give more width to `languages` side than `about me` side. Take a look at the both designs and fix your project to produce what is on the Figma template.

#### Your Design

![your-about-section](https://user-images.githubusercontent.com/26597432/200881829-0fee167e-1979-428a-ad70-6ada951c77ff.png)

### figma Design
![figma-about-section](https://user-images.githubusercontent.com/26597432/200881871-303e7df2-2e4f-41cb-b04a-f0e0c9f9e55a.png)